editor/src/clj/editor/resource_dialog.clj

```diff
@@ -0,0 +1,140 @@
+(ns editor.resource-dialog
+  (:require [editor.resource :as resource]
+            [dynamo.graph :as g]
+            [editor.workspace :as workspace]
+            [cljfx.fx.h-box :as fx.h-box]
+            [editor.ui :as ui]
```

Comment:
Nit-pick: unsorted requires.

Reply:
Good find, missed that! I'll fix it after vacation :)